bin/rundownloadgithub.sh

```diff
@@ -12,99 +12,61 @@
 # License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
 # either express or implied.
 
-# TODO use a function rather repeating code
-# TODO avoid git init / pull when not needed
-# TODO fix the flattening
+declare -A REPOSITORIES
+REPOSITORIES=([yangmodels/yang]=https://github.com/YangModels/yang.git \
+              [openconfig/yang]=https://github.com/openconfig/yang.git \
```

Comment:
This seems not to be used anymore and can be removed: `[openconfig/yang]=https://github.com/openconfig/yang.git`